### ISSUE TITLE
no-nested-interactive: false-positive when `<summary>` is directly inside `<details>`

### ISSUE THREAD

**Issue by ursm** [ember-template-lint/ember-template-lint#3251]
`ember-template-lint` v7.5.0 reports "Do not use `<summary>` inside `<details>`" for the canonical HTML pattern:

```hbs
<details>
  <summary>Toggle details</summary>
  More information…
</details>
```

Under the HTML spec `<summary>` must be the first child of a `<details>` element and is expressly allowed to be interactive.
The warning therefore looks like a false positive of the `no-nested-interactive` rule.

## Reproduction

1. Install ember-template-lint ^7.5.0
2. Create `app/templates/demo.hbs` with the snippet above
3. Run ember-template-lint

### Output:

```
  error  Do not use <summary> inside <details>  no-nested-interactive  demo.hbs:2:3
```

## Expected result
No lint error.

## Actual result
`no-nested-interactive` flags the pair as a violation.

**Comment by lifeart** [comment]
@ursm thank you for report! Issue fixed in https://github.com/ember-template-lint/ember-template-lint/pull/3249
